app/src/main/res/layout/activity_session_feedback.xml

```diff
@@ -23,234 +23,295 @@
                 app:popupTheme="@style/ThemeOverlay.AppCompat.Light" />
```

Comment:
I'm not a huge fun of this idea because it increases the hierarchy of layout.
I mean, I have to add another LinearLayout because `layout` tag and `ScrollView` can only have one children layout...